bueno/tools/lint.rs

```diff
@@ -0,0 +1,88 @@
+use deno_core::error::AnyError;
+use glob::glob;
+use oxc_allocator::Allocator;
+use oxc_ast::AstKind;
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_parser::Parser;
+use oxc_semantic::SemanticBuilder;
+use oxc_span::{SourceType, Span};
+use std::ffi::OsStr;
+use std::path::Path;
+
+pub struct LintOptions<'a> {
+    pub glob: &'a String,
+}
+
+pub fn lint(options: LintOptions) -> Result<bool, AnyError> {
+    let mut no_errors = false;
+
+    for entry in glob(&options.glob)? {
+        match entry {
+            Ok(path) => match path.extension().and_then(OsStr::to_str) {
+                Some("js" | "ts" | "jsx" | "tsx") => {
+                    no_errors = lint_file(path.as_path())? || no_errors;
+                }
+                _ => {}
+            },
+            Err(e) => println!("{:?}", e),
+        }
+    }
+
+    Ok(no_errors)
+}
+
+fn lint_file(path: &Path) -> Result<bool, AnyError> {
```

Comment:
we should return custom enum here like
```
enum LinterResult {
  Ok,
  Errors(Vec<Error>)
}
```
so we can properly handle errors later in one place instead of just printing them here, this looks untidy.